spec/order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'order'
+
```

Comment:
Since you're using dependency injection, you could double the takeaway in these tests.